setup.py

```diff
@@ -6,16 +6,23 @@
     packages=['mycroft_voice_satellite', 'mycroft_voice_satellite.speech',
               'mycroft_voice_satellite.res',
               'mycroft_voice_satellite.configuration'],
-    install_requires=["jarbas_hive_mind>=0.10.3",
-                      "speech2text",
+    install_requires=["jarbas_hive_mind",
+                      "SpeechRecognition @ git+https://github.com/Uberi/speech_recognition.git",
                       "pyee",
                       "requests",
                       "requests_futures",
                       "psutil",
-                      "PyAudio==0.2.11",
-                      "pocketsphinx==0.1.15",
+                      "PyAudio",
+                      "pocketsphinx",
+                      "json_database",
+                      "jarbas_utils",
                       "ovos_utils",
-                      "text2speech>=0.1.9"],
+                      "speech2text==0.2.1",
+                      "text2speech @ git+https://github.com/Joanguitar/text2speech.git",
```

Comment:
both speech2text and text2speech will be deprecated soon in favor of using https://github.com/OpenVoiceOS/OVOS-plugin-manager

why do you need to install from that branch?

Reply:
text2speech is set to work with Linux folders, I added a OS check to replace the temp folder with a Windows one if the OS is Windows.

Reply:
I should probably try to PR this repo first, but since you said it will be deprecated I don't see the point, hopefully we can eliminate it soon.